src/main/java/uk/nhs/digital/nhsconnect/nhais/model/fhir/NhsIdentifier.java

```diff
@@ -11,6 +13,9 @@
 
     public NhsIdentifier(String nhsNumber) {
         super();
+        if (StringUtils.isBlank(nhsNumber)) {
+            throw new FhirValidationException("Nhs number cannot be blank");
```

Comment:
Is this needed / doing anything now that validator has been added to the translator?

Reply:
you're right